Bottom: df0418b04a3a730965e919a398f28921eb417e2a
Top:    d47ba7a4c4d0a59f988b67cb0743d6243785c30c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-21 23:44:50 +0000

Make --enable-dbus default


---

diff --git a/configure.ac b/configure.ac
index 08ffa53..863e2ee 100644
--- a/configure.ac
+++ b/configure.ac
@@ -353,7 +353,13 @@ fi
 AC_MSG_CHECKING([for whether to use DBUS])
 AC_ARG_ENABLE([dbus],
 [  --enable-dbus           Enable DBUS IPC],
-[],[enable_dbus=no])
+[],[
+	case " $with_gui " in
+		*\ gtk\ *)     enable_dbus=yes ;;
+		*\ lesstif\ *) enable_dbus=yes ;;
+		* )            enable_dbus=no ;;
+	esac
+])
 
 AC_MSG_RESULT([$enable_dbus])
 AM_CONDITIONAL(WITH_DBUS, test x$enable_dbus = xyes)
